docker-compose.yaml

```diff
@@ -13,4 +13,4 @@ services:
         no_proxy:
         socks_proxy:
     ports:
-      - '3000:3000'
\ No newline at end of file
+      - '8080:3000'
```

Comment:
@DorTruzman Did you changed the port to 8080? If so, it supposed to be 8080:8080

Reply:
Changed to 8080